dict.js

```diff
@@ -14,7 +14,7 @@ function Dict(values, getDefault) {
     }
     getDefault = getDefault || Function.noop;
     this.getDefault = getDefault;
-    this.store = {};
+    this.store = Object.create(null);
```

Comment:
Note that this change takes as farther away from cross engine compatibility, not closer, regardless of the lack of work gone into validating this library across a spectrum of browser versions.